### DISCUSSION TITLE
Add option to remove sidebar title

### DISCUSSION THREAD

**Discussion by rocketeeer** [Able-Bear-Software/tab-nodes-tree#48] [Ideas / Feature Request]
Something nice-to-have would be an option to remove the title from the tab side-bar to reclaim some screen space.  I think it would be reasonable to include this as premium-only feature requiring a license.

**Comment by zeroarst** [comment]
Unfortunately, this is another thing that controlled by Chrome not developers. 😥 Some users also ask for reducing side panel size. But those cannot be changed by developers. I would suggest to raise a ticket in https://groups.google.com/a/chromium.org/g/chromium-extensions. If anyone knows any solution. Please let me know.
